helm/values.yaml

```diff
@@ -13,7 +12,7 @@ envVars:
 image:
   repository: tykio/tyk-operator
   pullPolicy: IfNotPresent
-  tag: "latest"
+  tag: "v0.9.0"
```

Comment:
Before cutting a release, we should update this tag according to new release version, right?

Reply:
Yeah. We will automate that in githubAction in separate ticket.